libs/redisconsumer/consumer_test.go

```diff
@@ -0,0 +1,56 @@
+package redisconsumer
+
+import (
+	must "github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestChunkMessages(t *testing.T) {
```

Comment:
It's good to have a test. However, I can suggest two improvements:
1. Do not reuse the `testMessages` variable. That's a well-known thing. And while reusing it here does not cause any meaningful harm, it does not bring any good at the same time. So let's don't do that. Unfortunately, this code base has got many of such examples, and adding more does not look good to me.
2. We've got multiple test cases here, but everything is coupled together under the single test. I strongly suggest, at least, using `t.Run` for each scenario. Supposedly, there should be a few test cases:
    - 0 elements
    - under the limit elements;
    - over the limit elements.